src/DuckLogger.h

```diff
@@ -31,7 +31,7 @@
 
 
 // https://github.com/esp8266/Arduino/blob/65579d29081cb8501e4d7f786747bf12e7b37da2/cores/esp8266/Print.cpp#L50
-static size_t cdpPrintf(const char *format, ...) {
```

Comment:
the cdpPrintf is actually used. It is how the logger macro are implemented if you check just down below this code
```
#define logerr(format, ...)                                     \
  do {                                                          \
    cdpPrintf("[E]");                                           \
    cdpPrintf("[** %s : %d] ",__FILENAME__, __LINE__);          \
    cdpPrintf(format, ##__VA_ARGS__);                           \
  } while (0)